### PR TITLE
added ability to change due date

### PR DESCRIPTION
**Pull request by jeremymeyers**
## Description

Added the ability to change due date to a todoist-formatted thing.  Defaults to "Tomorrow" as before

## Type of change

- [X] Improvement of an existing script

## Screenshot

n/a

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/todoist/create-task.template.sh [hunk 1]
```diff
@@ -25,14 +26,23 @@ if [ -z "$API_TOKEN" ]; then
...
+        	--data '{"content": "'"$TASK"'", "due_string": "'"$DUE"'"}' \
+        	-H "Content-Type: application/json" \
+        	-H "Authorization: Bearer $API_TOKEN"
+	else
+	    curl -s "https://api.todoist.com/rest/v1/tasks" \
+        	-X POST \
+        	--data '{"content": "'"$TASK"'", "due_string": "Tomorrow"}' \
+        	-H "Content-Type: application/json" \
+        	-H "Authorization: Bearer $API_TOKEN"
+	fi
```
Comment (unnamedd) [inline commands/apps/todoist/create-task.template.sh:44]: You don’t need to duplicate the whole request, just change the ‘due_string’ with the due date or ‘tomorrow’.

**Comment by jeremymeyers** [comment]
not sure the best way to do that? (sorry just getting into coding shell scripts)

**Comment by unnamedd** [comment]
> not sure the best way to do that? (sorry just getting into coding shell scripts)

You already did! :D

But check it out here:
```bash
DUE="Tomorrow"
if [[ $2 != "" ]]; then
    DUE=$2
fi

if [[ $TASK != "" ]]; then
    curl -s "https://api.todoist.com/rest/v1/tasks" \
    	-X POST \
    	--data '{"content": "'"$TASK"'", "due_string": "'"$DUE"'"}' \
    	-H "Content-Type: application/json" \
    	-H "Authorization: Bearer $API_TOKEN"

	echo "Task Created" # These tasks will show up in your inbox
else
	echo "Please specify a task"
fi
```

**Comment by jeremymeyers** [comment]
Ahh yes that makes sense! will revise and patch

**Review by mattisssa — APPROVED** [review]
Looks good! Thanks for the PR and updates

**Pull request merged**